services/gps.md

```diff
@@ -0,0 +1,58 @@
+# GPS
+
+    identifier: 0x19dd6136
+    extends: _sensor
+    camel: gps
+
+A Global Positioning Device. NMEA message reference at https://www.sparkfun.com/datasheets/GPS/NMEA%20Reference%20Manual-Rev2.1-Dec07.pdf.
+
+## Registers
+
+    ro position @ reading {
+        timestamp: u32 utc_time
```

Comment:
I think we should have millisecond-precision time here; maybe u64?

Reply:
The serial interface reports seconds.

Reply:
adafruit guide has this example `$GPRMC,194509.000,A,4042.6142,N,07400.4168,W,2.03,221.11,160412,,,A*77` where the time is `19:45:09.000`; it is `.000` but I think it could have more precision; GPS does have a very precise time